core/kazoo_modb/priv/couchdb/views/services.json

```diff
@@ -10,7 +10,143 @@
     "language": "javascript",
     "views": {
         "audit_logs_by_creation": {
-            "map": "function(doc) { if ( doc.pvt_deleted || doc.pvt_type != 'audit_log') return; emit(doc.pvt_created, {id: doc._id, status: doc.status, reason: doc.reason, message: doc.message, type: doc.audit.changes.type, summary: doc.invoice.summary}); }"
+            "map": [
+                "function(doc) {",
+                "  if ( doc.pvt_deleted || doc.pvt_type != 'audit_log') return;",
+                "  emit(doc.pvt_created, {",
+                "    'id': doc._id,",
+                "    'status': doc.status,",
+                "    'reason': doc.reason,",
+                "    'message': doc.message,",
+                "    'type': doc.audit.changes.type,",
+                "    'summary': doc.invoice.summary",
+                "  });",
+                "}"
+            ]
+        },
+        "day_summary_by_date": {
+            "map": [
+                "function(doc) {",
+                "  if (doc.pvt_deleted || (doc.pvt_type != 'audit_log' && doc.pvt_type != 'ledger')) return;",
+                "  var unix = new Date(Math.floor((doc.pvt_created - 62167219200) * 1000));",
+                "  var day = unix.getUTCFullYear() + '-' + ('0' + (unix.getUTCMonth() + 1)).slice(-2) + '-' + ('0' + unix.getUTCDate()).slice(-2);",
+                "  if (doc.pvt_type === 'audit_log') {",
+                "    var invoice = doc.invoice || {};",
+                "    var items = invoice.items || [];",
+                "    for (var index = 0; index < items.length; index++) {",
+                "      var item = items[index];",
+                "      if (item.changes && (item.changes.difference.billable || 0 ) !== 0) {",
+                "        var changedBillable = item.changes.difference.billable;",
+                "        var result = {};",
+                "        result[item.item] = {",
+                "          'addition': changedBillable > 0 ? changedBillable : 0,",
+                "          'last_timestamp': doc.pvt_created,",
+                "          'quantity': item.billable || 0,",
+                "          'removal': changedBillable < 0 ? Math.abs(changedBillable) : 0,",
+                "          'sum_quantity': false",
+                "        };",
+                "        emit(day, result);",
+                "      }",
+                "    }",
+                "  } else if ((doc.pvt_type === 'ledger') && (doc.source.service === 'per-minute-voip')) {",
```

Comment:
is there a reason to only include per-minute-voip ledger ?

Reply:
because the UI needs to show per-minute-voip, and audit log is not generated for per-minute.

Reply:
what the UI shows should not determine the api, i don't see a reason not to include other ledgers here.